src/content/developers/docs/scaling/validium/index.md

```diff
@@ -7,30 +7,66 @@ incomplete: true
 sidebarDepth: 3
 ---
 
-Uses validity proofs like [ZK-rollups](/developers/docs/scaling/zk-rollups/) but data is not stored on the main layer 1 Ethereum chain. This can lead to 10k transactions per second per validium chain and multiple chains can be run in parallel.
+Validium is a [scaling solution](/developers/docs/scaling/) that enforces integrity of transactions using validity proofs like [ZK-rollups](/developers/docs/scaling/zk-rollups/), but doesn’t store transaction data on the Ethereum Mainnet. While off-chain data availability introduces trade-offs, it can lead to massive improvements in scalability (validiums can process ~ 9,000 transactions, or more, per second). 
 
 ## Prerequisites {#prerequisites}
 
 You should have a good understanding of all the foundational topics and a high-level understanding of [Ethereum scaling](/developers/docs/scaling/). Implementing scaling solutions such as Validium is an advanced topic as the technology is less battle-tested, and continues to be researched and developed.
 
-## Pros and cons {#pros-and-cons}
+## How does validium work? {#how-does-validium-work}
 
+Validiums are a type of “rollup”: an off-chain scaling protocol designed to improve throughput by processing transactions off the Ethereum Mainnet. Off-chain transactions executed on the validium chain are verified via a smart contract on the base Ethereum layer using zero-knowledge proofs, which can be SNARKs (Succinct Non-Interactive Argument of Knowledge) or STARKs (Scalable Transparent ARgument of Knowledge). 
+
+More on [zero-knowledge proofs](https://consensys.net/blog/blockchain-explained/zero-knowledge-proofs-starks-vs-snarks/).
+
+As with ZK-rollups, validity proofs prevent invalid state transitions on validium chains and enhance security guarantees available to users. For instance, a malicious operator cannot steal funds since users can always withdraw funds directly from the on-chain contract [using Merkle proofs](/developers/tutorials/merkle-proofs-for-offline-data-integrity/). 
+
+The major difference between validiums and ZK-rollups concerns their positions on the data availability spectrum. Both solutions approach data storage differently, which has implications for security and trustlessness. 
+
+### Off-chain vs on-chain data availability {#off-chain-vs-on-chain-data-availability}
```

Comment:
```suggestion
### On-chain vs off-chain data availability {#on-chain-vs-off-chain-data-availability}
```
Switching order since we open up with zk-rollups here

Reply:
I think in this section we could categorize different off-chain data availability solutions similar to how we do use cases on the bridges page (https://ethereum.org/en/bridges/#bridge-use-cases)

Something like:
```
### on-chain vs off-chain data availability
zk-rollup onchain blurb
validium offchain blurb

#### Data availability committees (DAC)
DAC blurb with some resources

#### Bond data availability
blurb on bond data availability with some resources

Off-chain data availability does present a problem: data necessary for creating Merkle proofs may be unavailable. This means users may be unable to withdraw funds from the rollup contract if operators should act maliciously. 
```